Set shape of initial BBH inspiral excisions to ID excisions

nilsvu opened the pull request:
## Proposed changes

Fixes #6126. This makes sure that the ID can always be interpolated to the Ev grid. Leaves #6128 open to do later.

### Upgrade instructions

<!--
If this PR makes changes that other people should be aware of when upgrading
their code, describe what they should do between the two UPGRADE INSTRUCTIONS
lines below.
-->
<!-- UPGRADE INSTRUCTIONS -->

<!-- UPGRADE INSTRUCTIONS -->

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### Further comments

<!--
If this is a relatively large or complex change, kick off the discussion by
explaining why you chose the solution you did and what alternatives you
considered, etc...
-->

nilsvu committed:
Set shape of initial BBH inspiral excisions to ID excisions

nilsvu commented on support/Pipelines/Bbh/Inspiral.yaml:
```diff
@@ -116,7 +116,7 @@ DomainCreator:
             - {{ ExcisionAShapeSpin_z }}
 {% endif %}
-        SizeInitialValues: [0.0, 0.0, 0.0]
+        SizeInitialValues: [0.0, -0.1, 0.0]
```
@knelli2 I had to set this as well to get the control systems to start. Do you have a better idea how to do this; and do you know what SpEC does here? @markscheel do you know?

geoffrey4444 replied:
Sometimes you have to do this in spec for high spins (make the excision surface move outward to avoid initial incoming char speeds), but I did not have to do this for the spec initial data I evolved

nilsvu replied:
I dropped this change and instead made the ID excisions bigger to match the value SpEC uses for negative-expansion boundary conditions. That seems to help.

nilsvu committed:
BBH ID: increase size of excision

Helps the evolution control system to get started initially.
SpEC seems to use a fraction of 0.97, but our control systems
can't shrink the horizons yet (need state 3). A fraction of 0.93
seems enough to get the control systems started.
